packages/fluent-theme/src/fluent/styles/Persona.styles.ts

```diff
@@ -0,0 +1,33 @@
+import { IPersonaStyleProps, IPersonaStyles, PersonaSize, sizeBoolean } from 'office-ui-fabric-react/lib/Persona';
+import { FontSizes } from '../FluentType';
+import { FontWeights } from 'office-ui-fabric-react/lib/Styling';
+import { NeutralColors } from '../FluentColors';
+
+export const PersonaStyles = (props: IPersonaStyleProps): Partial<IPersonaStyles> => {
+  const size = sizeBoolean(props.size as PersonaSize);
+
+  return {
+    primaryText: [
+      size.isSize100 && {
+        fontSize: FontSizes.size20,
+        fontWeight: FontWeights.regular
+      },
+      size.isSize72 && {
+        fontSize: FontSizes.size20
+      },
+      size.isSize48 && {
+        fontSize: FontSizes.size16
+      }
+    ],
+    secondaryText: {
+      color: NeutralColors.gray120
+    },
+    tertiaryText: {
+      fontSize: FontSizes.size14,
+      color: NeutralColors.gray120
```

Comment:
Not asking for a change, just thinking out loud--it would be really nice if we could use grays from the theme to avoid using "off-palette" colors.